src/renderers/dom/shared/ReactDOMEventListener.js

```diff
@@ -138,12 +137,12 @@ var ReactDOMEventListener = {
    */
   trapBubbledEvent: function(topLevelType, handlerBaseName, element) {
     if (!element) {
-      return null;
+      return;
     }
-    return EventListener.listen(
-      element,
+    element.addEventListener(
```

Comment:
The only case in which this wouldn't be equivalent is if `element` is truthy but not a DOM node. Doesn't seem like this could happen.

Reply:
Would it make sense to throw if `element` is falsy? I don't think there's any current code path where `trapBubbledEvent` or `trapCaptureEvent` should be called without a node.

Reply:
I think EventListener gets shimmed in www?

Reply:
There exists a mobile version that wraps into `TimeSlice` guards but we don't use it. At least not since flat bundles. Nobody complained.

Reply:
I'm curious about when `element` is null here too and would +1 an invariant or an action item to investigate this.

Reply:
I manually checked the callers and I can’t see how it possibly can be null. It always corresponds to `domNode` from either `ReactDOMComponent` or `ReactDOMFiberComponent`.

Reply:
I don't think an invariant is very valuable here because it's a leaf code path (throws immediately with a very obvious message).

Reply:
@gaearon: see https://fburl.com/8jv38orh

Reply:
Aah, I get it now. We leave `fbjs/lib/EventListener` import in the www flat bundle, which actually redirects to the internal `EventListener` fork.

Reply:
@gaearon what throws immediately? If this is protected from null elements earlier in the code path could we just remove the falsy check?

Reply:
@aweary Yes I think we can.